src/scripts/components/Tooltip.jsx

```diff
@@ -195,7 +171,19 @@ class Tooltip extends React.Component {
    * @return {void}
    */
   openTooltip(tooltip) {
-    tooltip.timeline.play();
+    const { delay } = this.props;
+
+    // Convert delay to a number
+    // Takes care of multiple PropTypes
+    const delayDuration = Number.isInteger(delay) ? Number(delay) : 1000;
+
+    if (delay) {
```

Comment:
It might be clearer, if you put the `delayDuration` calculation inside this if statement since it's the only place you use it.

Reply:
👍